api/database/factories/UserFactory.php

```diff
@@ -19,7 +19,7 @@
 */
 
 $factory->define(User::class, function (Faker $faker) {
-    $email = $faker->unique()->safeEmail;
+    $email = "schedulia-test@schedulia.xyz";
```

Comment:
I am not sure we should hardcode only one email here. We should be able to seed N amount of users.